src/docs.mts

```diff
@@ -1,84 +1,183 @@
-import { remark } from 'remark';
-import type { Code, Root } from 'mdast';
-import { readFileSync, writeFileSync, mkdirSync, existsSync } from 'fs';
+/**
+ * @overview Process and potentially transform documentation source files into files suitable for publishing.
+ *   Copy files from the source directory (/src/docs) to the directory used for the final, published documentation (/docs).
+ *   The list of files changed (transformed) and copied to /docs are logged to the console.
+ *   If a file in /src/docs has the same contents in /docs, nothing is done (it is not copied to /docs).
+ *
+ * @example docs
+ * @example docs --verify
+ *   If the --verify option is used, no files will be copied to /docs, but the list of files to be changed is still shown on the console.
+ *   A message to the console will show that this command should be run without the --verify flag so that the 'docs' folder is be updated.
+ *   Note that the command will return an exit code (1), which will show that it failed.
+ * @example docs --git
+ *   If the --git option is used, the command `git add docs` will be run
+ *
+ */
+
+import { remark } from "remark";
+import type { Code, Root } from "mdast";
+import { readFileSync, writeFileSync, mkdirSync, existsSync } from "fs";
+import { JSDOM } from "jsdom";
+
+// @ts-ignore
+import flatmap from "unist-util-flatmap";
+import { globby } from "globby";
+import { join, dirname } from "path";
+import { exec } from "child_process";
 // @ts-ignore
-import flatmap from 'unist-util-flatmap';
-import { globby } from 'globby';
-import { join, dirname } from 'path';
-import { exec } from 'child_process';
-import prettier from 'prettier';
```

Comment:
<img width="198" alt="image" src="https://user-images.githubusercontent.com/10703445/188543372-c054f6ff-6ada-4583-a3ec-0ebd4d2bd18a.png">

Can you run prettier on this? It should be single quotes.

Reply:
will do.

(I think it's because I'm just using eslint to format, and that defaults to double quotes)

Reply:
We should figure out why the prettier precommit hook didn't fix it. 
Did it not run when committing?

Reply:
Aha.  I turned off the githooks when I was working on something else.  That must be it.
I'm sure it's something on my side (my IDE preferences, probably).